packages/experimental/src/experimental-list/task-item.tsx

```diff
@@ -128,11 +130,19 @@ export const TaskItem: React.FC< TaskItemProps > = ( {
 		( ( onDismiss || remindMeLater ) && ! completed ) ||
 		( onDelete && completed );
 
+	const toggleActionVisibility = () => {
+		setTaskExpanded( ! isTaskExpanded );
```

Comment:
Should we still provide an optional callback here? `onExpand` & `onCollapse`? 
Then also add a `useEffect` that listens to `expanded` prop and updates the `isTaskExpanded` when `expanded` is updated?

Reply:
> Should we still provide an optional callback here? onExpand & onCollapse?

This was added in the commit 8c06f57

> Then also add a `useEffect` that listens to `expanded` prop and updates the `isTaskExpanded` when `expanded` is updated?

I addressed this change in the commit d108345